MapboxNavigation/FeedbackSubtypeCollectionViewCell.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  FeedbackSubtypeCollectionViewCell.swift
+//  MapboxNavigation
+//
+//  Created by Avi Cieplinski on 8/17/20.
+//  Copyright © 2020 Mapbox. All rights reserved.
+//
+
+import UIKit
+
+class FeedbackSubtypeCollectionViewCell: UICollectionViewCell {
+    static let defaultIdentifier = "MapboxFeedbackSubtypeCell"
+
+    struct Constants {
+        static let circleSize: CGSize = 36.0
+        static let imageSize: CGSize = 36.0
+        static let padding: CGFloat = 30
+        static let titleFont: UIFont = .systemFont(ofSize: 18, weight: .semibold)
+    }
+
+    lazy var circleView: UIView = .forAutoLayout()
+
+    lazy var separatorView: UIView = .forAutoLayout()
+
+    lazy var titleLabel: UILabel = {
+        let title: UILabel = .forAutoLayout()
+        title.numberOfLines = 2
+        title.textAlignment = .center
+        title.font = Constants.titleFont
+        return title
+    }()
+
+    public var circleColor: UIColor = .black {
+        didSet {
+            circleView.backgroundColor = circleColor
+        }
+    }
+
+    public var circleOutlineColor: UIColor = .black {
+        didSet {
+            circleView.layer.borderColor = circleOutlineColor.cgColor
+        }
+    }
+
+    public var separatorColor: UIColor = .lightGray {
+        didSet {
+            separatorView.backgroundColor = separatorColor
+        }
+    }
+
+    var longPress: UILongPressGestureRecognizer?
+    var originalTransform: CGAffineTransform?
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        commonInit()
+    }
+
+    func commonInit() {
+        setupViews()
+        setupConstraints()
+    }
+
+    override var isHighlighted: Bool {
+        didSet {
+            if originalTransform == nil {
+                originalTransform = self.circleView.transform
+            }
+
+            UIView.defaultSpringAnimation(0.3, animations: {
+                if self.isHighlighted {
+                    self.circleView.transform = self.circleView.transform.scaledBy(x: 0.85, y: 0.85)
+                } else {
+                    guard let t = self.originalTransform else { return }
+                    self.circleView.transform = t
```

Comment:
```suggestion
                    guard let transform = self.originalTransform else { return }
                    self.circleView.transform = transform
```

Reply:
We can't use `transform` as it resolves to the view's own `transform` property, but to follow the spirit of your comment I changed `t` to `originalTransform`.